src/routes.js

```diff
@@ -358,12 +355,6 @@ module.exports.bind = function (app) {
   account.get(paymentTypes.index, simplifiedSettingsRedirect, permission('payment-types:read'), paymentTypesController.getIndex)
   account.post(paymentTypes.index, simplifiedSettingsRedirect, permission('payment-types:update'), paymentTypesController.postIndex)
 
-  // MOTO mask card number & security code
-  account.get(toggleMotoMaskCardNumberAndSecurityCode.cardNumber, simplifiedSettingsRedirect, permission('moto-mask-input:read'), toggleMotoMaskCardNumber.get)
-  account.post(toggleMotoMaskCardNumberAndSecurityCode.cardNumber, simplifiedSettingsRedirect, permission('moto-mask-input:update'), toggleMotoMaskCardNumber.post)
-  account.get(toggleMotoMaskCardNumberAndSecurityCode.securityCode, simplifiedSettingsRedirect, permission('moto-mask-input:read'), toggleMotoMaskSecurityCode.get)
-  account.post(toggleMotoMaskCardNumberAndSecurityCode.securityCode, simplifiedSettingsRedirect, permission('moto-mask-input:update'), toggleMotoMaskSecurityCode.post)
```

Comment:
not strictly part of this PR but I noticed the new MOTO security settings just have `payment-types:update` permissions, so maybe worth changing those to use these `moto-mask-input:[read/update]` permissions`

Reply:
Yeah, lets do that as seperate PR